### PR TITLE
create model

### PR DESCRIPTION
**Pull request by PhanThanhGiang91**

### CODE REVIEW

File: app/models/category.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class Category < ActiveRecord::Base
+	has_many :words
```
Comment (chuanhtuan) [inline app/models/category.rb]: indent 2

File: app/models/lesson.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class Lesson < ActiveRecord::Base
+	has_many :lesson_words
```
Comment (chuanhtuan) [inline app/models/lesson.rb]: indent 2

File: app/models/lesson.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+	has_many :words, :through => :lesson_words
```
Comment (chuanhtuan) [inline app/models/lesson.rb]: has_many :words, through :lesson_words

File: app/models/lesson_word.rb [hunk 4]
```diff
@@ -0,0 +1,6 @@
+class LessonWord < ActiveRecord::Base
+
+	belongs_to :word
```
Comment (chuanhtuan) [inline app/models/lesson_word.rb]: indent 2

File: db/migrate/20150617031506_create_lesson_words.rb [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+  def change
+    create_table :lesson_words do |t|
+
+    	t.belongs_to :lesson, index: true
+    	t.belongs_to :word, index: true
+    	t.belongs_to :answer, index: true
+      t.timestamps null: false
+    end
+  end
+end
```
Comment (khanhkhanh) [inline db/migrate/20150617031506_create_lesson_words.rb]: Break line

**Pull request merged**